pkg/controller/controller.go

```diff
@@ -475,7 +475,11 @@ func (p *csiProvisioner) ProvisionExt(options controller.ProvisionOptions) (*v1.
 		case pvcKind:
 			rc.clone = true
 		default:
-			klog.Infof("Unsupported DataSource specified (%s), the provisioner won't act on this request", options.PVC.Spec.DataSource.Kind)
+			klog.Infof("DataSource specified (%s) is not supported by the provisioner, waiting for an external data populator to create the volume", options.PVC.Spec.DataSource.Kind)
+			// DataSource is not VolumeSnapshot and PVC
+			// Wait for an external data populator to create the volume
+			//controller.eventRecorder.Event(options.PVC, v1.EventTypeNormal, "Provisioning", fmt.Sprintf("Waiting for a volume to be created by an external data populator"))
```

Comment:
should we add the event? Even if it's not the complete solution, it's more end-user facing than a log.

Reply:
Yeah, I'd like to add an event.  I just realized that eventRecorder is defined in the sig-storage-lib-external-provisioner controller, but not in the provisioner controller.  So I'll add a new eventRecorder param in the provisioner controller.

Reply:
Done